### PR TITLE
New test procedure for TigerVNC version >= 1.12

### PR DESCRIPTION
**Pull request by pablo-herranz**
According to this [tutorial](https://github.com/TigerVNC/tigervnc/blob/master/unix/vncserver/HOWTO.md), the tests for TigerVNC versions greater than or equal to 1.12.0 must be redefined.

- Related ticket: https://progress.opensuse.org/issues/107248
- Needles:
- Verification runs:
    [Tumbleweed](https://openqa.opensuse.org/tests/overview?distri=opensuse&version=Tumbleweed&build=poo107320_investigation_tumbleweed)
    [SLE15 SP4](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=poo107320_investigation_sle15sp4)

### CODE REVIEW

File: tests/x11/vnc_two_passwords.pm [hunk 1]
```diff
@@ -145,10 +158,21 @@ sub run {
     send_key "ctrl-c";
     wait_still_screen 2;
 
+    # Check tigervnc version. Test will differ for versions under 1.12
+    my $tigervnc_vers = script_output("rpm -q tigervnc | awk -F \'-\' \'{print \$2}\'");
```
[the remark this answers is not in the document]
Comment (pablo-herranz) [reply tests/x11/vnc_two_passwords.pm]: Changed now :)

**Comment by foursixnine** [comment]
@pablo-herranz can you schedule couple of runs? something like 10 for o3 and for openSUSE?

**Comment by pablo-herranz** [comment]
> @pablo-herranz can you schedule couple of runs? something like 10 for o3 and for openSUSE?

Done, they're running now. (link in description :))

**Comment by DimStar77** [comment]
> > @pablo-herranz can you schedule couple of runs? something like 10 for o3 and for openSUSE?
> 
> Done, they're running now. (link in description :) )

Have the SLE tests been flaky before? [SLE15 SP4](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=poo107320_investigation_sle15sp4) lists two failures

**Comment by pablo-herranz** [comment]
> > > @pablo-herranz can you schedule couple of runs? something like 10 for o3 and for openSUSE?
> > 
> > 
> > Done, they're running now. (link in description :) )
> 
> Have the SLE tests been flaky before? [SLE15 SP4](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=poo107320_investigation_sle15sp4) lists two failures

Looks like they're failing in a step non-related with my code. Anyway, I'll schedule some more without my changes and see.

**Comment by pablo-herranz** [comment]
Runs without changes here --> https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=poo107320_investigation_sle15sp4_nochanges

**Comment by pablo-herranz** [comment]
> > > @pablo-herranz can you schedule couple of runs? something like 10 for o3 and for openSUSE?
> > 
> > 
> > Done, they're running now. (link in description :) )
> 
> Have the SLE tests been flaky before? [SLE15 SP4](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=poo107320_investigation_sle15sp4) lists two failures

Yup, looks like it was failing before my changes --> https://openqa.suse.de/tests/8455880#step/vnc_two_passwords/33
I've scheduled another 25 to see what happens.

**Pull request merged**